charts/ctlog-tiles/README.md

```diff
@@ -0,0 +1,193 @@
+# ctlog-tiles
+
+<!-- This README.md is generated. Please edit README.md.gotmpl -->
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.1.1-0.20251112113307-52b154e44576](https://img.shields.io/badge/AppVersion-v0.1.1--0.20251112113307--52b154e44576-informational?style=flat-square)
+
+Tiles-based certificate log (TesseraCT)
+
+**Homepage:** <https://sigstore.dev>
+
+## Quick Installation
+
+To install the helm chart with default values run following command.
+The [Values](#values) section describes the configuration options for this chart.
+
+```shell
+helm dependency update .
+helm install [RELEASE_NAME] .
+```
+
+## Uninstallation
+
+To uninstall the Helm chart run following command.
+
+```shell
+helm uninstall [RELEASE_NAME]
+```
+
+## Specifying Fulcio roots
+
+Unlike the ctlog chart, there is no createctconfig job that runs and fetches Fulcio's roots from the rootCert endpoint. The roots need to be known ahead of time and specified directly in values.yaml:
+
+```
+server:
+  ...
+  fulcio:
+    rootPEM: |-
+      -----BEGIN CERTIFICATE-----
+      ...
+```
+
+## Using the POSIX backend
+
+In values.yaml, set `image.flavor` to `"posix"` (default).
+
+Create a secret for the private key:
+
+```
+kubectl -n ctlog-tiles-system create secret generic ctlog-signing-key --from-file=signing-key=/path/to/privkey.pem
```

Comment:
Can we give instructions on generating the private key too, just an openssl incantation?